bitnami/apache/README.md

```diff
@@ -44,41 +44,53 @@ The command removes all the Kubernetes components associated with the chart and
+| `image.repository`               | Apache Docker image name                                | `bitnami/apache`                                             |
+| `image.tag`                      | Apache Docker image tag                                 | `{TAG_NAME}`                                                 |
+| `image.pullPolicy`               | Apache Docker image pull policy                         | `Always`                                                     |
+| `image.pullSecrets`              | Specify Docker registry secret names as an array        | `[]` (does not add image pull secrets to deployed pods)      |
+| `git.registry`                   | Git image registry                                      | `docker.io`                                                  |
+| `git.repository`                 | Git image name                                          | `bitnami/git`                                                |
+| `git.tag`                        | Git image tag                                           | `{TAG_NAME}`                                                 |
+| `git.pullPolicy`                 | Git image pull policy                                   | `Always`                                                     |
+| `git.pullSecrets`                | Specify docker-registry secret names as an array        | `[]` (does not add image pull secrets to deployed pods)      |
+| `replicaCount`                   | Number of replicas of the Apache deployment             | `docker.io`                                                  |
+| `htdocsConfigMap`                | ConfigMap with the server static content                | `nil`                                                        |
+| `htdocsPVC`                      | PVC with the server static content                      | `nil`                                                        |
+| `cloneHtdocsFromGit.enabled`     | Get the server static content from a git repository     | `false`                                                      |
+| `cloneHtdocsFromGit.repository`  | Repository to clone static content from                 | `nil`                                                        |
+| `cloneHtdocsFromGit.branch`      | Branch inside the git repository                        | `nil`                                                        |
+| `cloneHtdocsFromGit.interval`    | Interval for sidecar container pull from the repository | `60`                                                         |
+| `podAnnotations`                 | Pod annotations                                         | `{}`                                                         |
+| `ingress.enabled`                | Enable ingress controller resource                      | `false`                                                      |
```

Comment:
I'm missing the `resources`, `livenessProbe` and `readinessProbe` parameters information which are present in the **values.yaml**